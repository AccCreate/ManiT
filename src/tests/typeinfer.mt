i = 1;
f = 1.11;
s = "String Test";
bt = true;
bf = false;

print(i);
print(f);
print(s);
print(bt);
print(bf);