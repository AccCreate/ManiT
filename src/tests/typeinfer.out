1
String Test
1
0
